### DISCUSSION TITLE
Intermediate sync with OpenSSL 1.1.1?

### DISCUSSION THREAD

**Discussion by baentsch** [open-quantum-safe/openssl#372] [Polls]
It looks like several test certs in OpenSSL expired on Jun 1 - which lets the upstream OpenSSL test suite fail now (also explaining [recent CI failures here](https://app.circleci.com/pipelines/github/open-quantum-safe/openssl/992/workflows/73ddf0ab-7f03-46fe-8a49-04817c46c955/jobs/4827)).

**Comment by christianpaquin** [comment]
If the cherry picking is simple enough, that might be a good option.

**Reply by dstebila** [reply]
It seems like there are relatively few commits on the 1_1_1-stable branch beyond the ones we'd cherry pick, so it might be about as easy to sync with all the recent commits as it is to cherry-pick just the 4 of interest.

**Reply by baentsch** [reply]
Opted for cherry pick: #373
